src/data/reference/hi.json

```diff
@@ -196,7 +218,11 @@
             "returns": "p5 ऑब्जेक्ट"
         },
         "fill": {
-            "description": "ज्यामितीय आंकड़े भरने के लिए उपयोग किए जाने वाले रंग को परिभाषित करता है। उदाहरण के लिए, यदि आप भरण (204, 102, 0) चलाते हैं, तो नीचे दिए गए सभी आंकड़ों में नारंगी भरण होगा। यह रंग RGB या HSB रंग के संदर्भ में निर्दिष्ट है, colorMode () के अनुसार रंग मोड पर निर्भर करता है (डिफ़ॉल्ट रंग डोमेन RGB है, प्रत्येक मान 0 और 255 के बीच की सीमा में है)। यदि कोई स्ट्रिंग प्रकार तर्क दिया जाता है, तो RGB, RGBA और हेक्साडेसिमल CSS प्रकार समर्थित हैं। भराव के रंग को परिभाषित करने के लिए एक कलर पी 5 ऑब्जेक्ट प्रदान किया जा सकता है।",
+            "description": [
+                "आकृतियों को भरने के लिए प्रयुक्त रंग सेट करता है। उदाहरण के लिए, यदि आप भरण (204, 102, 0) चलाते हैं, तो भरण आदेश के बाद खींची गई सभी आकृतियों को नारंगी रंग से भर दिया जाएगा। यह रंग या तो आरजीबी के संदर्भ में निर्दिष्ट है या वर्तमान <a href=\"#/p5/colorMode\">colorMode()</a> के आधार पर HSB रंग। (डिफ़ॉल्ट रंग स्थान RGB है, जिसमें प्रत्येक मान 0 से 255 के बीच होता है)। अल्फा डिफ़ॉल्ट रूप से सीमा भी 0 से 255 तक होती है।",
```

Comment:
Don't translate the function names into hindi please ;

Reply:
ok